test/unit/core/config/type-cast/index.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * @prettier
+ */
+import typeCast from "core/config/type-cast"
+
+jest.mock("core/presets/apis", () => {})
```

Comment:
I had to mock this import from default options:
```js
import ApisPreset from "core/presets/apis"
```
which somewhere down the line tries to import from `swagger-client` and other modules which threw errors in the test.